Guesser/AbstractTypeGuesser.php

```diff
@@ -11,9 +11,9 @@
 
 namespace Sonata\PropelAdminBundle\Guesser;
 
+use Propel\Bundle\PropelBundle\Form\PropelTypeGuesser;
 use Sonata\AdminBundle\Guesser\TypeGuesserInterface;
 use Sonata\AdminBundle\Model\ModelManagerInterface;
-use Symfony\Bridge\Propel1\Form\PropelTypeGuesser;
```

Comment:
I didn't expect this PR will be accepted blindly after the first submit (the travis config has not been updated yet), and it is ok for the review.

I just got the feeling that our project is the only one which use this bundle since the BC break of february. 
And the last sucessful build is 2 years old: https://travis-ci.org/sonata-project/SonataPropelAdminBundle/builds. So, I'm still thinking the risk is limited, considering this bundle is not stable. 

Anyway, I agree to have a sf 2.8 requirement and still have to push the new Travis config (to have a successful build for instance ;)).